src/components/question/index.js

```diff
@@ -1,15 +1,17 @@
 import React, {useEffect,useState} from "react";
 import Heading from "../question/Header/Heading";
 import advenPic from "../../assets/adventure.svg";
-import Answers from "./Answer.jsx";
 import styles from '../../App.module.css';
 
 
-const Question = ({question, incorrectAnswers, correctAnswer}) => {
+const Question = ({incrementIndex,incrementScore,question, incorrectAnswers, correctAnswer}) => {
     let [answer,setAnswers] = useState([]);
     useEffect(()=>{
         setAnswers(handleshuffle(correctAnswer, incorrectAnswers));
-    },[])// dependency array run one time
+    },[correctAnswer, incorrectAnswers]) // dependency array run one time
+    function Answers({answers}){ // make it one component
+        return <li>{answers}</li>
+    }
```

Comment:
you can directly return the <li> line 27

Reply:
done